Xamarin.Forms.Core/MenuItem.cs

```diff
@@ -6,6 +6,12 @@ namespace Xamarin.Forms
 {
 	public class MenuItem : BaseMenuItem, IMenuItemController
 	{
+		public static readonly BindableProperty AcceleratorProperty = BindableProperty.CreateAttached("Accelerator", typeof(string), typeof(MenuItem), null);
```

Comment:
I was thinking about the future of this API a bit and I think we need to be slightly more clever.

public class Accelerator { // name not good
  public static Accelerator FromString (string accel);
}

This way if we ever need to add an enum (god forbid) based API in the future we can.